extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1092,6 +1088,8 @@ export class AmpA4A extends AMP.BaseElement {
     const iframe = /** @type {!HTMLIFrameElement} */(
         createElementWithAttributes(
             /** @type {!Document} */(this.element.ownerDocument), 'iframe', {
+              height: this.creativeSize_.height,
```

Comment:
Sure, then I suppose setting auto has no effect?  Important thing is to document that it can occur and the result.  Thanks